infra/cifuzz/cifuzz.py

```diff
@@ -524,6 +524,14 @@ def is_project_sanitizer(sanitizer, oss_fuzz_project_name):
 
   # Simple parse to prevent adding pyYAML dependency.
   with open(project_yaml, 'r') as file_handle:
-    if sanitizer + '\n' in file_handle.read():
-      return True
+    file_data = file_handle.read()
+
+  # TODO(metzman): Replace this with proper handling of project.yaml files.
+  if 'sanitizers:\n' not in file_data:
+    logging.info('No sanitizers defined in project.yaml. '
+                 'Only allowing address and undefined')
```

Comment:
"Default to" would be confusing and misleading in the context of CIFuzz because it defaults to 'address'. The wording looks good to me considering it's just supposed to make it clear that either 'address' or 'undefined' are considered valid at that point.